DependencyInjection/TrsteelCkeditorExtension.php

```diff
@@ -43,39 +43,9 @@ public function load(array $configs, ContainerBuilder $container)
             'Cache.SerializerPath' => '%kernel.cache_dir%',
         ), $config['html_purifier']['config']);
 
+        $container->setParameter('trsteel_ckeditor.ckeditor', $config);
```

Comment:
It would probably be a good idea to unset the unnecessary parameters (e.g $config['class'] and $config['html_purifier']).